cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -118,10 +118,14 @@ func (m *awsWrapper) getManagedNodegroupInfo(nodegroupName string, clusterName s
 		taintList := r.Nodegroup.Taints
 		for _, taint := range taintList {
 			if taint != nil && taint.Effect != nil && taint.Key != nil && taint.Value != nil {
+				formattedEffect, err := taintEksTranslator(taint)
+				if err != nil {
+					return nil, labels, tags, err
```

Comment:
If there's an error, I generally feel like everything else should be nil -- prevents partial data bugs in the future.

Reply:
A valid and good point. Thanks